go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/onc-healthit/lantern-back-end
 
-go 1.14
\ No newline at end of file
+go 1.14
+
+require github.com/onc-healthit/lantern-back-end/endpointmanager v0.0.0-20210325125748-e07becdde8ac // indirect
```

Comment:
Any idea why this file was updated?

Reply:
Not sure, probably a mistake while updating mods. I just reverted it back to the original so it should be all set now!

Reply:
sounds good!